Header header
int64 new_max_speed
int64 new_max_range
bool start